node/src/test/scala/com/wavesplatform/http/AssetsRouteSpec.scala

```diff
@@ -309,13 +312,43 @@ class AssetsRouteSpec extends RouteSpec("/assets") with Eventually with RestAPIS
     checkDetails(d, route, issue, issue.id().toString, assetDescr)
   }
 
-  routePath(s"/details/{id} - non-smart asset") in routeTest() { (d, route) =>
-    val tx = issueTransaction()
+  routePath(s"/details/{id} - non-smart asset") in withDomain(RideV6, AddrWithBalance.enoughBalances(defaultSigner)) { d =>
```

Comment:
Maybe it would be better to support `balances` argument for `routeTest()` function?
This will prevent code duplication for `AssetsApiRoute` creation